app/src/validators/team.validator.js

```diff
@@ -11,8 +11,9 @@ class TeamValidator {
         ctx.checkBody('layers').optional();
         const isArrayToErrors = (field) => {
           const value = ctx.request.body[field];
-          if (typeof value !== 'undefined' && !Array.isArray(value)) {
-            ctx.errors = [ ...ctx.errors , { [field]: `${field} should be an Array.` }];
+          if (value !== undefined && !Array.isArray(value)) {
```

Comment:
As a convention we prefer maintaining `typeof value !== 'undefined'`. This case the other condition will work fine, but its a little bit tedious to check every time in code reviews whether it's correct or not. So prefer `typeof` always. ✌️